integration-tests/plugins/__tests__/price-lists/admin/delete-price-list-spec.ts

```diff
@@ -0,0 +1,145 @@
+import { useApi } from "../../../../environment-helpers/use-api"
+import { getContainer } from "../../../../environment-helpers/use-container"
+import { initDb, useDb } from "../../../../environment-helpers/use-db"
+import {
+  simpleProductFactory,
+  simpleRegionFactory,
+} from "../../../../factories"
+
+import { IPricingModuleService } from "@medusajs/types"
+import path from "path"
+import { startBootstrapApp } from "../../../../environment-helpers/bootstrap-app"
+import adminSeeder from "../../../../helpers/admin-seeder"
+import { createDefaultRuleTypes } from "../../../helpers/create-default-rule-types"
+import { createVariantPriceSet } from "../../../helpers/create-variant-price-set"
+
+jest.setTimeout(50000)
+
+const adminHeaders = {
+  headers: {
+    "x-medusa-access-token": "test_token",
+  },
+}
+
+const env = {
+  MEDUSA_FF_MEDUSA_V2: true,
+}
+
+describe("[Product & Pricing Module] DELETE /admin/price-lists/:id", () => {
```

Comment:
I don't think it changes anything much, I can spend some time doing it if you insist but I don't think individual files for each endpoint is harming the readability of the code + it's how endpoint tests are structured in the core

Reply:
not sure it is how it is structured in the core integration tests, example integration-tests/api/__tests__/admin/products/ff-product-categories.js among all the other example :)

Reply:
Sorry for the misunderstanding,  i was referring to the core endpoint tests. 

I think dividing files by endpoint is a reasonable way of structuring these rather than grouping them, I think this way provides better readability of each test rather than mixing in a couple of different endpoints in each file.